packages/docs/src/pages/components/progress/examples/ExValues.vue

```diff
@@ -5,7 +5,7 @@
             <b-field label="Type and Size">
                 <b-field grouped>
                     <b-select v-model="type" placeholder="Type">
-                        <option value="null">Default</option>
+                        <option>Default</option>
```

Comment:
This implies the `value` attribute is not `undefined` but `"Default"`.
![image](https://github.com/user-attachments/assets/40b7547d-f07b-40c5-9f6d-c213592182f2)

Reply:
We have to explicitly give `undefined` here.
```html
<option :value="undefined">Default</option>
```

Reply:
I initially thought your idea was good, but the actual behavior was not what we expected.